catalog/app/containers/Bucket/viewModes.spec.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * @jest-environment jsdom
```

Comment:
i think we should set this globally in `jest.config.js`, since we're testing a web app and targeting browser-like environment